go-controller/pkg/ovn/common.go

```diff
@@ -21,6 +21,35 @@ func hashedAddressSet(s string) string {
 	return fmt.Sprintf("a%s", hashString)
 }
 
+func (oc *Controller) forEachAddressSetUnhashedName(iteratorFn func(
+	string, string, string)) error {
+	// forEachAddressSetUnhashedName will pass the unhashedName, namespaceName and
```

Comment:
Please move the function description above the function.

Reply:
oops, put it in wrong place. Moved in new ps.